core/src/main/java/org/frankframework/configuration/filters/InitialCapsFilter.java

```diff
@@ -42,7 +42,7 @@ public void endElement(String uri, String localName, String qName) throws SAXExc
 	}
 
 	protected String initialCap(String elementName) {
-		return Character.toUpperCase(elementName.charAt(0))+elementName.substring(1);
+		return StringUtil.ucFirst(elementName);
```

Comment:
Lijkt me helemaal prima hoor, ik heb alleen een vraag: waarom gebruiken voor dit soort dingen niet apache StringUtils? in dit geval bv `org.apache.commons.lang3.StringUtils#capitalize`?

Reply:
Performance reasons :)